src/buildMathML.js

```diff
@@ -137,22 +159,29 @@ export const buildExpression = function(expression, options) {
  * Equivalent to buildExpression, but wraps the elements in an <mrow>
  * if there's more than one.  Returns a single node instead of an array.
  */
-export const buildExpressionRow = function(expression, options) {
+export const buildExpressionRow = function(
+    expression: AnyParseNode[],
+    options: Options,
+): MathDomNode {
     return makeRow(buildExpression(expression, options));
 };
 
 /**
  * Takes a group from the parser and calls the appropriate groupBuilders function
  * on it to produce a MathML node.
  */
-export const buildGroup = function(group, options) {
+export const buildGroup = function(
+    group: ?AnyParseNode,
+    options: Options,
+): MathDomNode {
     if (!group) {
         return new mathMLTree.MathNode("mrow");
     }
 
     if (groupBuilders[group.type]) {
         // Call the groupBuilders function
-        const result = groupBuilders[group.type](group, options);
+        // $FlowFixMe
+        const result: MathDomNode = groupBuilders[group.type](group, options);
```

Comment:
Why `$FlowFixMe`?

Reply:
There's no way in the type system to say that `groupBuilders[group.type]` has type `MathMLBuilder<TYPE>` where `group.type` (dynamic) `=== TYPE` (compile-time).

Reply:
But isn't `groupBuilders` an array of `MathMLBuilder`, which is a function returns `MathDomNode`? So if `groupBuilders[group.type]` isn't undefined, wouldn't `groupBuilders[group.type](group, options)` be `MathDomNode`?

Reply:
Sorry for the confusion: the return to isn't the issue here, even though it was written out explicitly; it's the calling of the function (`groupBuilders[group.type](group, options)`) with the given `group` that requires the `$FlowFixMe`.